utils/validate-install-plan.js

```diff
@@ -0,0 +1,77 @@
+'use strict';
+
+const { fetchPaginatedGHResults } = require('./github-api-helpers');
+const { findMainInstallConfigFiles, readQuickstartFile } = require('./helpers');
+const path = require('path');
+
+const url = process.argv[2];
```

Comment:
I wonder if we should standardize to the approach used [here in another script in the repo](https://github.com/newrelic/newrelic-quickstarts/blob/main/utils/sanitize_dashboards.js#L3-L4):

```
// Removes the first two arguments passed by `node`, these aren't needed for this function to run - CM
const pathsToSanitize = process.argv.slice(2);
```
Maybe an addition to the helpers utility file?